recreate-dependabot-pr

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/python3
+
+# This file is part of Cockpit.
+#
+# Copyright (C) 2024 Red Hat, Inc.
+#
+# Cockpit is free software; you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation; either version 2.1 of the License, or
+# (at your option) any later version.
+#
+# Cockpit is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+
+# Update COCKPIT_REPO_COMMIT to cockpit HEAD automatically, defaults to
+# Makefile as input optionally the full path can be provided. (For example
+# Anaconda uses ui/webui/Makefile.am).
+
+import os
+import sys
+import time
+from typing import Optional
+
+import task
+from lib.aio.jsonutil import get_dict, get_int, get_str
+from task import github
+
+sys.dont_write_bytecode = True
+
+
+def main() -> None:
+    assert os.getenv('GITHUB_BASE') is not None, 'GITHUB_BASE must be set'
+
+    api = github.GitHub()
+    for pull in api.pulls(state="open"):
+        number = get_int(pull, 'number')
+        if number is None:
+            continue
```

Comment:
:eyes: how can this happen?

Reply:
It can't be we are going full mypy so we have to check for this.

Reply:
I see!